pkg/services/config.go

```diff
@@ -356,7 +357,7 @@ func New(ctx context.Context, config Config) (*Services, error) {
 		authenticators = union.New(authenticators, authn.NewNoAuth(gatewayClient))
 	}
 
-	if config.EmailServerName != "" {
+	if config.EmailServerName != "" && config.EnableSMTPServer {
```

Comment:
we are likely going to change the implementation for how we should match the email server name for sendgrid, but right now it follows the same pattern with smtp approach. So we need to set the email server name without having to setup smtp server. This is a way to do it specifically through a different envariable.